### DISCUSSION TITLE
Resetting errors

### DISCUSSION THREAD

**Discussion by hjaved-nha** [edmundhung/conform#366] [Q&A] [answered]
Hi 👋🏼 ,
Is there a way to reset all validation errors ? I have a modal form and would like to clear all the validation errors once the model form is closed. If I open the modal form again, the validation errors are shown on the form still.

React hook form has [reset](https://www.react-hook-form.com/api/useform/reset/) and [clearErrors](https://react-hook-form.com/docs/useform/clearerrors)  which can be invoked when the modal is closed. Does conform has such functionality ?

Thanks

**Comment by edmundhung** [comment, marked as answer]
You can trigger a native form `reset` event, which conform listens to, using `formElement.reset()`. If you are not passing a custom ref object to `useForm()`, you should have a default form ref object with `form.ref` already.

**Reply by hjaved-nha** [reply]
Hey,

Thanks for the prompt reply. That worked 🎉 . Should be documented (unless I missed it 😅) ?
